app/controllers/user_sessions_controller.rb

```diff
@@ -1,9 +1,15 @@
 class UserSessionsController < ApplicationController
-  before_filter :reset_session, only: [:destroy]
-  before_filter :require_no_user, :only => [:new, :create]
-  before_filter :require_user, :only => :destroy
+  before_action :reset_session, only: [:destroy]
+  before_action :require_no_user, only: %w[new create]
+  before_action :require_user, only: :destroy
 
-  def security_notification; end
+  def new
+    construct_user_session
+  end
+
+  def security_notification
+    redirect_to(account_path) if current_user
+  end
 
   def create
```

Comment:
`create` is one of several methods in this controller that do not appear to be in use anymore. We should purge those, but we should also ensure that the logic in those methods is implemented somewhere or captured in a story (if still needed). For example, `redirection_path` determines which page a user should land on after logging in, which is not currently implemented anywhere. We should review that and determine if it's still necessary. Also, the `construct_user_session`, which is no longer in use, included a line to secure the session:
```
@user_session.secure = Rails.application.config.ssl_options[:secure_cookies]
```
That seems like something we need to retain, as it ensures that the cookie will only be sent over SSL connections.

Reply:
so I saw that too. But was afraid taking it out would break something.

Reply:
yeah we don't need to keep the redirection_path. Omniauth will return that. See https://github.com/omniauth/omniauth/wiki/Saving-User-Location.  There is already a jira ticket to ensure user continues to land on the page they were trying to visit after logging in. That would address this. 

I was going to not touch much of this unused code  but I'll try to remove lets see what happens if its going to be time suck to cleanup rspec tests that are tied to this, I will suggest we wait till after ATO.